transportation-data-publishing/data_tracker/signal_pms_fulcrum.py

```diff
@@ -250,6 +258,13 @@ def replace_pm_records(
         postgre_records_df["fulcrum_id"].isin(knack_pm_records_df["FULCRUM_ID"])
     ].copy()
 
+    # # get the current year
```

Comment:
Should we drop this commented out code? Or will you need it in the future?

Reply:
Good catch, I've deleted that part, thanks!